### DISCUSSION TITLE
How about adding feature for static webhosting [ build ]

### DISCUSSION THREAD

**Discussion by ostrichegret** [denchiklut/ssr-boilerplate#17] [Ideas]
How about adding a feature for static web hosting like [create-react-app] build?
or React Static Site Generator

Thank you

**Comment by denchiklut** [comment]
> How about adding a feature for static web hosting like [create-react-app] build?
> 
> or React Static Site Generator
> 
> 
> 
> Thank you

Sorry I somehow missed this message 🙈
Can u explain a bit more about the feature?
